api/descriptions/collections/post.md

```diff
@@ -1,23 +1,28 @@
-Creates a new collection.
-Any field left unspecified is assigned the value of `""` (an empty string).
+Creates a new collection for use in rules and views across the product.
+
+To invoke this endpoint in the Console UI:
+
+1. Navigate to **Manage > Collections and Tags > Collections**.
+2. Add a collection using **+ Add collection**.
+3. Click the **Save** button.
+
+### cURL Request
+
+The following cURL command creates a new collection.
 
 ```bash
-$ curl -k \
+$ curl 'https://<CONSOLE>/api/v1/collections' \
```

Comment:
@KICKdesigns I'd like our curl commands to be more "concrete". In other words, you just need to replace a few `<VARS>`, and you've got a fully working curl command that you run and get a valid result. I want customers to be able to easily tweak and build on the curl commands we provide.

With that in mind, let's do this in the command:

* For collection name, call it `my-collection`.
* Create a collection that captures all ubuntu:18.04 images. Set all other fields as wildcards.
* Provide a short description for the curl command. What is it doing? What are the significant parameters? Example: you're creating a collection named `my-collection`, which captures all container images named ubuntu:18.04.

Reply:
@KICKdesigns I'm playing with this endpoint now, and I just realized that unspecified resources are assigned a wildcard by default. I'll fix the curl command now.

Reply:
Fixed in my latest commit.